sys-apps/qdirstat/qdirstat-1.2.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="A cross-platform graphical disk usage utility"
+DESCRIPTION="QDirStat is a graphical application to show \
+where your disk space has gone and to help you clean it up. \
+QDirStat is the Qt-only port of the old Qt3/KDE3-based KDirStat now \
+based on the latest Qt5. It runs on every X11-based desktop on \
+Linux, BSD, and other Unix-like systems. \
+QDirStat has a number of new features compared to KDirStat: \
+* Multi-selection in both the tree and the treemap.
+* Unlimited number of user-defined cleanup actions.
+* Properly show errors of cleanup actions.
+* File categories and their treemap color are now configurable.
+* Exclude rules for directories are easily configurable.
+* Cross-Platform; independent on KDE or any other specific toolkit.
+"
+HOMEPAGE="https://github.com/shundhammer/qdirstat"
+COPYRIGHT="2015, 2017 Stefan Hundhammer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/shundhammer/qdirstat/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="d79734915768ae858c784142c7a07b7599178329b40c73894bf575decaf1a911"
+PATCHES="qdirstat-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86 ?x86_64"
+
+PROVIDES="
+	qdirstat$secondaryArchSuffix = $portVersion
+	app:QDirStat$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	qt5$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	qt5${secondaryArchSuffix}_devel
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:qmake$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	"
+BUILD()
+{
+	qmake .
+	make $jobArgs
+}
+INSTALL()
+{
+	mkdir -p $appsDir/QDirStat
+	cp src/qdirstat $appsDir/QDirStat/QDirStat
```

Comment:
A folder for a single binary is discouraged. Just copy the binary in $appsDir, not a subdirectory.